_for_authors/module_updates.md

```diff
@@ -0,0 +1,29 @@
+# Module Updates
+
+As an author, you create a high quality module and contribute it to our project.  That's a generous and important donation of time and materials.  As time goes on, almost every module will need updating.  This is inevitable and does not reflect lack of commitment or quality in the original material.
+
+Updating can take place because of changes to the topic.  For example: software is updated and code samples or screenshots need to be changed, better methods emerge and should be included, or trends and usage of particular libraries or packages change over time.  Updates can also happen because of external factors that don't have to do with the topic *per se*. Linked articles cease to exist and need to be replaced, for instance, or the infrastructure that allows for in-browser coding changes.  A module may also need to be updated to account for bugs or other quality issues.  Quiz questions might be tweaked to reflect feedback from learners, for example, or typos or broken links fixed.
+
+This need for updating and maintenance on an ongoing basis can prove problematic for modules contributed by authors who are not core staff who work together daily.  You may be too busy to make suggested changes, or your interests may have changed and you're just not interested in continuing to donate your labor to continual improvement.  We may not have the most up-to-date contact information for you and might not even be able to reach you to inform you about desired changes.
+
+## Updates We Initiate
```

Comment:
```suggestion
## Updates You Initiate

If at any point you wish to suggest a change to a module you wrote, please let us know! You can send us an email at dart@chop.edu, or [post an issue](https://github.com/arcus/education_modules/issues/new/choose) to let us know what should be changed. 

We will take your suggestions into account and use them at our discretion.  If at any point the module diverges from your vision and you prefer that the attribution be changed, you can request us to change the attribution to be a co-authorship between you and DART staff or ask us to make this attribution a generic "DART contributor" attribution and remove your name from the module. 

## Updates We Initiate
```

Reply:
In a meeting we talked about including a brief section on author-initiated changes as well, to make it clear we welcome that.